microsoft-edge/devtools-guide-chromium/javascript/snippets.md

```diff
@@ -22,149 +22,158 @@ ms.date: 05/04/2021
 
-   ![The Show Snippets command](./snippets-images/javascript-search-show-snippets.png)
+   ![The Show Snippets command](./snippets-images/command-menu.png)
 
 
 <!-- ====================================================================== -->
-## Create Snippets
+## Create new snippets
 
-### Create a Snippet through the Sources tool
+To create a new snippet from the **Snippets** tab:
 
 1. [Open the Snippets tab](#open-the-snippets-tab).
 
 1. Click **New snippet**.
 
-1. Enter a name for your Snippet, and then press **Enter**:
+1. Enter a name for your snippet, and then press **Enter**:
 
-   ![Name a Snippet](./snippets-images/javascript-sources-snippets-naming.png)
+   ![A new, empty, snippet in the Sources tool](./snippets-images/new-snippet.png)
 
-### Create a Snippet through the Command Menu
+#### Create a new snippet with the Command Menu
 
 1. Focus your cursor somewhere in DevTools.
 
 1. Press **Ctrl+Shift+P** (Windows, Linux) or **Command+Shift+P** (macOS) to open the Command Menu.
 
-1. Start typing **snippet**, select **Create new snippet**, and then press **Enter**:
+1. Start typing "snippet", select **Create new snippet**, and then press **Enter**:
 
-   ![The command for creating a new Snippet](./snippets-images/javascript-search-create-new-snippet.png)
+   ![The command for creating a new snippet](./snippets-images/new-snippet-command-menu.png)
 
-To rename your new Snippet with a custom name, see [Rename Snippets](#rename-snippets).
+To rename your new snippet, see [Rename snippets](#rename-snippets).
 
 
 <!-- ====================================================================== -->
-## Edit Snippets
+## Edit snippets
+
+To edit the source code of a snippet:
 
 1. [Open the Snippets tab](#open-the-snippets-tab).
 
-1. In the **Snippets** tab, select the name of the Snippet that you want to edit.  The Snippet opens in the **Code Editor**:
+1. In the **Snippets** tab, click the name of the snippet that you want to edit.  The snippet opens in the **Code Editor**:
 
-   ![The Code Editor](./snippets-images/javascript-sources-snippets-editor-saved.png)
+   ![The Code Editor](./snippets-images/edit-snippet.png)
 
-1. Use the **Code Editor** to add JavaScript to your Snippet.
+1. Use the **Code Editor** to add JavaScript to your snippet.
 
-1. When an asterisk appears next to the name of your Snippet, it means you have unsaved code.  Press **Ctrl+S** (Windows, Linux) or **Command+S** (macOS) to save:
+1. When an asterisk appears next to the name of your snippet, it means you have unsaved code.  Press **Ctrl+S** (Windows, Linux) or **Command+S** (macOS) to save:
 
-   ![An asterisk next to the Snippet name indicates unsaved code](./snippets-images/javascript-sources-snippets-editor-unsaved.png)
+   ![An asterisk next to the snippet name indicates unsaved code](./snippets-images/unsaved-changes.png)
 
 
 <!-- ====================================================================== -->
-## Run Snippets
+## Run snippets
```

Comment:
```suggestion
## Run a snippet
```
Global: prefer singular to plural.  The task/activity/scenario/ use-case is running 1 snippet, not running multiple snippets.